integration-tests/test_register.py

```diff
@@ -55,6 +56,28 @@ def test_register_with_org(external_candlepin, subman, test_config):
                                {},
                                locale)
     assert subman.is_registered
+
+
+@pytest.mark.parametrize("enable_content",["true","false"])
+def test_register_with_enable_content(external_candlepin, subman, test_config, enable_content): 
+    """
+    https://www.candlepinproject.org/docs/subscription-manager/dbus_objects.html#methods-6
+    """
+    assert not subman.is_registered
+    proxy = RHSM.get_proxy(RHSM_REGISTER_SERVER)
+    with RHSMPrivateBus(proxy) as private_bus:
+        private_proxy = private_bus.get_proxy(RHSM.service_name,
+                                              RHSM_REGISTER.object_path)
+        locale = os.environ.get("LANG", "")
+        private_proxy.Register(test_config.get("candlepin","org"),
```

Comment:
Just suggestion, calling this D-Bus method should return some object and it could be used for testing too. This is also true for other D-Bus methods.

Reply:
done - I will add more assertation in next PR to cover this suggestion.